tests.js

```diff
@@ -188,6 +188,30 @@ describe('Bro.makeItHappen', function() {
     });
 });
 
+describe('Bro.setItUp', function() {
+    var success,
+        expected,
+        obj;
+
+    beforeEach(function() {
+        obj = { "foo": { "bar": {} } };
+    });
+
+    it('should set existing deeply nested properties on an object', function() {
+        expected = { "foo": { "bar": 'awesome' } };
+        var bro = Bro(obj);
+        bro.setItUp('foo.bar', 'awesome');
+        assert.deepEqual(expected, obj);
+    });
+
+    it('should create new properties, then set them, as needed', function() {
+        expected = { "foo": { "bar": { "stuff": { "and": { "things": 'super awesome' } } } } };
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Too many errors. (94% scanned).